src/solver/SolverLayout.tsx

```diff
@@ -15,7 +15,8 @@ import {
   useNodesState,
   useReactFlow,
 } from '@xyflow/react';
-import { useEffect, useRef, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import { EnterFullScreenIcon, ExitFullScreenIcon } from '@radix-ui/react-icons';
```

Comment:
We are already using `@tabler/icons` for icons, I'd suggest `IconWindowMaximize` or `IconArrowMaximize` maybe?

![image](https://github.com/user-attachments/assets/53219b81-2c34-4241-9dfa-7a940bfb577e)
![image](https://github.com/user-attachments/assets/072f540d-5e2d-4d55-b207-3b3e530056c7)